config/routes.rb

```diff
@@ -0,0 +1,63 @@
+Rails.application.routes.draw do
+  resources :accounts do
+    resources :features, module: :accounts
+    resources :messages, module: :accounts
+    resources :routes, module: :accounts
+  end
+  root 'accounts#index'
```

Comment:
Pretty simple app. CRUD for accounts + namespaced features, messages and routes.

It was very very weird actually having a model called "Route" given how richly associated that already is in Rails-land.